pbjam/peakbag.py

```diff
@@ -374,59 +372,4 @@ def __call__(self, model_type='simple', tune=1500, nthreads=1, maxiter=4,
         except:
             self.summary = pm.stats.summary(self.samples)
             
-        self.par_names = self.summary.index
```

Comment:
This is a large deletion - is it really related to adding plotting functionality?

Reply:
Nope. Unrelated to plotting.py I just spotted it when I was adding some attributes to peakbag.